src/utils/fetchPeopleData.ts

```diff
@@ -0,0 +1,12 @@
+export async function fetchPeopleData() {
+  try {
+    const response = await fetch(
+      'https://mate-academy.github.io/react_people-table/api/people.json',
```

Comment:
Avoid using hardcoded URLs. It's better to store URLs in a separate config file or at least as a constant at the top of the current file.